src/vector/index.js

```diff
@@ -49,6 +48,7 @@ import UAParser from 'ua-parser-js';
 import url from 'url';
 
 import {parseQs, parseQsFromFragment} from './url_utils';
+import IntegrationManager from './integration';
```

Comment:
We should probably call these something other than 'IntergrationManager' as that's going to confuse people with scalar. How about `PlatformManager` (or just `Platform`)?